src/oom/oomd-util.c

```diff
@@ -145,7 +145,7 @@ bool oomd_swap_free_below(const OomdSystemContext *ctx, int threshold_permyriad)
 }
 
 int oomd_fetch_cgroup_oom_preference(OomdCGroupContext *ctx, const char *prefix) {
-        uid_t uid, prefix_uid;
+        uid_t uid = UID_INVALID, prefix_uid = UID_INVALID;
```

Comment:
It is not necessary to initialize `uid`, as `cg_get_owner()` always assign a value on success.